m4/Makefile.am

```diff
@@ -22,4 +22,12 @@
 
 EXTRA_DIST = \
     as-version.m4 \
+    gettext.m4 \
+    iconv.m4 \
+    lib-ld.m4 \
+    lib-link.m4 \
+    lib-prefix.m4 \
+    nls.m4 \
+    po.m4 \
+    progtest.m4 \
```

Comment:
I'd think to commit those files in the patch if we append those files to EXTRA_DIST.

Reply:
I don't think they need to be stored in git, they are copied by `autopoint`.